SerialPrograms/Source/PokemonSV/Programs/General/PokemonSV_GimmighoulChestFarmer.cpp

```diff
@@ -107,19 +107,33 @@ void GimmighoulChestFarmer::program(SingleSwitchProgramEnvironment& env, BotBase
         pbf_move_left_joystick(context, 0, 0, 158, 0);
         pbf_press_button(context, BUTTON_L, 50, 40);
         pbf_move_left_joystick(context, 128, 0, 100, 0);
-        //snake eater
+        //Climb ladder
         pbf_press_button(context, BUTTON_L, 50, 40);
         pbf_move_left_joystick(context, 128, 0, 2350, 0);
         pbf_press_button(context, BUTTON_L, 50, 40);
-        pbf_move_left_joystick(context, 128, 0, 140, 0);
+        pbf_wait(context, 100);
+        context.wait_for_all_requests();
+        //Walk into the wall
+        pbf_move_left_joystick(context, 128, 0, 200, 0);
+        pbf_wait(context, 100);

```

Comment:
You can merge away these waits:
`pbf_move_left_joystick(context, 128, 0, 200, 100);`